File: 백준/Bronze/9325. 얼마？/얼마？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int testCase = sc.nextInt();

        for (int i = 0; i < testCase; i++) {
            int carPrice = sc.nextInt();
            int optionCase = sc.nextInt();
            int sum = carPrice;

            for (int j = 0; j < optionCase; j++) {
                int optionNum = sc.nextInt();
                int optionPrice = sc.nextInt();
                sum += optionNum * optionPrice;
            }

            System.out.println(sum);
        }
    }
}

File: 백준/Bronze/9325. 얼마？/얼마？.cc
#include <iostream>
using namespace std;

int main() {
	int testCase;
	cin >> testCase;

	for (int i = 0; i < testCase; i++)
	{
		int sum = 0;
		int carPrice;
		cin >> carPrice;

		sum += carPrice;

		int optionNum;
		cin >> optionNum;

		for (size_t i = 0; i < optionNum; i++)
		{
			int n;
			int optionPrice;
			cin >> n >> optionPrice;

			sum += (n * optionPrice);
		}

		cout << sum << endl;
	}
}